src/utils/filter/getTimeForMongo.ts

```diff
@@ -13,7 +13,7 @@ export const getTimeForMongo = (value: any): Date => {
         timeZone: 'Europe/Berlin',
       })
     );
-  } else if (value.match(/^\d\d:\d\d$/)) {
+  } else if (value?.match(/^\d\d:\d\d$/)) {
```

Comment:
@AntoineRelief Just leaving this for your review. I am not sure if we want to keep this "?" or not. It prevents throwing an error when date objects are passed as parameters instead of strings, but I'm not sure we want to give that implicit flexibility to this function.